### PR TITLE
[docs] Mention our beloved sponsors in README.md

### PR DESCRIPTION
**Pull request by rtsisyk**

### CODE REVIEW

**Review by vng — APPROVED** [review]

**Comment by fitojb** [comment]
Happy to see the NLNet Foundation, a name I knew previously as they have sponsored some targeted LibreOffice development :)

File: README.md [hunk 1]
```diff
@@ -19,12 +19,6 @@ No ads, no tracking, no data collection, no crapware. Your [donations](https://o
   <a href="https://f-droid.org/en/packages/app.organicmaps/">
     <img alt="Get it on F-Droid" src="docs/badges/fdroid.png" width="180">
   </a>
-  <a href='https://flathub.org/apps/details/app.organicmaps.desktop'>
-    <img alt="Download on Flathub" src="docs/badges/flathub.png" width="180"/>
-  </a>
-  <a href="https://repology.org/project/organicmaps/versions">
```
Comment (pastk) [inline README.md:25]: but why remove flathub and repology links?

File: README.md [hunk 2]
```diff
@@ -102,11 +96,46 @@ Reject surveillance - embrace your freedom.
...
+      <a href="https://github.com/organicmaps/organicmaps/milestone/7">The Search & Fonts improvement project</a> has been <a href="https://nlnet.nl/project/OrganicMaps/>funded</a> through NGI0 Entrust Fund. <a href="https://nlnet.nl/entrust/">NGI0 Entrust Fund</a> is established by the <a href="https://nlnet.nl/">NLnet Foundation</a> with financial support from the European Commission's <a href="https://www.ngi.eu/">Next Generation Internet programme</a>, under the aegis of DG Communications Networks, Content and Technology under grant agreement No 101069594.
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <a href="https://summerofcode.withgoogle.com/"><img src="docs/sponsors/gsoc.svg" alt="Google Summer of Code" width="200px"></a>
+    </td>
+    <td>
+      <a href="https://summerofcode.withgoogle.com/">Google</a> backed 5 student's projects in the Google Summer of Code program during <a href="https://summerofcode.withgoogle.com/programs/2022/organizations/organic-maps">2022</a> and <a href="https://summerofcode.withgoogle.com/programs/2023/organizations/organic-maps">2023</a> programs. Noteworthy projects included Android Auto, Outdoor Style, and Wikipedia Dump Extractor.
```
Comment (pastk) [inline README.md:115]: Outdoor Style wasn't a GSOC project, so better remove it please.

(The project was about internal refactoring of the styles system, so not really user-facing, hence maybe little point including it in the list).

File: README.md [hunk 3]
```diff
@@ -102,11 +96,46 @@ Reject surveillance - embrace your freedom.
...
+    <td>
+      <a href="https://futo.org"><img src="docs/sponsors/futo.svg" alt="FUTO" width="200px"></a>
+    </td>
+    <td>
+      <a href="https://futo.org">FUTO</a> has <a href="https://www.youtube.com/watch?v=fJJclgBHrEw">awarded $1000 micro-grant</a> to Organic Maps in February 2023.
+    </td>
+  </tr>
+</table>
+
+The majority of all expenses have been funded by founders of the project since its inception. The project is far from achieving any sort of financial sustainability. The current level of voluntary donations falls significantly short of covering efforts needed to sustain the app. Any new developments of features are beyond the scope of possibility due to the absence of the necessary financial resources.
```
Comment (pastk) [inline README.md:136]: > Any new developments of features are beyond the scope of possibility due to the absence of the necessary financial resources.

Sounds too pessimistic / not true IMO. Actually there were newly developed features already and I'm pretty sure there will be more to come despite this fact of underfunding. I think better rephrase it like "underfunding slows down development of new features significantly".

@biodranik WDYT?

**Pull request merged**